sparse_ho/tests/test_models.py

```diff
@@ -88,6 +89,52 @@ def test_beta_jac_custom(model_name):
         assert np.allclose(jac, jac_custom)
 
 
+@pytest.mark.parametrize('model_name', list(custom_models.keys()))
+def test_warm_start(model_name):
+    """Check that using sk or celer yields the same solution as sparse ho"""
+    if model_name == "svm" or model_name == "svr" or model_name == "ssvr":
+        X_s = X_r
+    else:
+        X_s = X_c
+
+    for log_alpha in dict_list_log_alphas[model_name]:
+        model = models[model_name]
+
+        # Compute beta and jac a first time
```

Comment:
In fact I suggest doing :

```python
mask, dense, jac = None, None, None
for i in range(2):
    the duplicated block of code 
    and then if i == 1 assert that n_iter is 2
```
to avoid having the duplicated block that we have now